crates/egui/src/widget_text.rs

```diff
@@ -100,6 +101,28 @@ impl RichText {
         self
     }
 
+    /// Extra spacing between letters, in points.
+    ///
+    /// Default: 0.0. Round to whole _pixels_ for crisp text.
```

Comment:
Do you have a section on pixel vs. point in the docs you could link to from here?

Reply:
I'll add something!